linode/nb/tmpl/basic.gotf

```diff
@@ -5,6 +5,7 @@ resource "linode_nodebalancer" "foobar" {
     region = "{{ .Region }}"
     client_conn_throttle = 20
     tags = ["tf_test"]
+    transfer {}
```

Comment:
I was asked to add the `transfer` block here to compile the code, probably because I implemented the `transfer` as `ListNestedBlock` in the schema. Will it introduce a breaking change?

Reply:
Let me check if there is a possible workaround...

Reply:
Judging from the migration docs, it looks like we'll need to use a `ListAttribute` with a `types.ObjectType` for the `ElementType` field (similar to how we used to do data sources). 

See https://developer.hashicorp.com/terraform/plugin/framework/migrating/attributes-blocks/blocks-computed#framework